docs/tutorial/part-two/index.md

```diff
@@ -296,36 +292,36 @@ export default () => (
 
-The finished page should now look like:
+समाप्त पेज अब ऐसा दिखना चाहिए:
 
-![User list page with CSS modules](css-modules-userlist.png)
+![सीएसएस मॉड्यूल के साथ उपयोगकर्ता सूची पेज](css-modules-userlist.png)
 
 ### CSS-in-JS
 
-CSS-in-JS is a component-oriented styling approach. Most generally, it is a pattern where [CSS is composed inline using JavaScript](https://reactjs.org/docs/faq-styling.html#what-is-css-in-js).
+CSS-in-JS एक कौम्पोनॅन्ट-ओरियेनटेड स्टाइलिंग दृष्टिकोण है। ज्यादातर आम तौर पर, यह एक पैटर्न है जहां [सीएसएस जावास्क्रिप्ट का उपयोग करके इनलाइन बनाया जाता है](https://reactjs.org/docs/faq-styling.html#what-is-css-in-js)
 
-#### Using CSS-in-JS with Gatsby
+#### Gatsby के साथ CSS-in-JS का उपयोग करना
 
-There are many different CSS-in-JS libraries and many of them have Gatsby plugins already. We won't cover an example of CSS-in-JS in this initial tutorial, but we encourage you to [explore](/docs/styling/) what the ecosystem has to offer. There are mini-tutorials for two libraries, in particular, [Emotion](/docs/emotion/) and [Styled Components](/docs/styled-components/).
+कई अलग-अलग CSS-in-JS लाइब्रेरीज़ हैं और उनमें से कई में पहले से ही Gatsby प्लगइन्स मौजूद हैं। हम इस प्रारंभिक ट्यूटोरियल में CSS-in-JS के एक उदाहरण को शामिल नहीं करेंगे, लेकिन हम आपको इस बात का पता लगाने के लिए प्रोत्साहित करते हैं कि ईकोसिस्टम को क्या करना है। दो पुस्तकालयों के लिए मिनी-ट्यूटोरियल हैं, विशेष रूप से, [Emotion](/docs/emotion/) और [Styled Components](/docs/styled-components/)।
 
-#### Suggested reading on CSS-in-JS
+#### CSS-in-JS पर सुझाया गया सुझाव
 
-If you're interested in further reading, check out [Christopher "vjeux" Chedeau's 2014 presentation that sparked this movement](https://speakerdeck.com/vjeux/react-css-in-js) as well as [Mark Dalgleish's more recent post "A Unified Styling Language"](https://medium.com/seek-blog/a-unified-styling-language-d0c208de2660).
+यदि आप आगे पढ़ने में रुचि रखते हैं, तो [Christopher "vjeux" Chedeau की 2014 की प्रस्तुति देखें, जिसने इस आंदोलन को उकसाया] (https://speakerdeck.com/vjeux/react-css-in-js) और साथ ही साथ [मार्क डॉकलेश का हाल की पोस्ट ["A Unified Styling Language"](https://medium.com/seek-blog/a-unified-styling-language-d0c208de2660)
 
-### Other CSS options
+### अन्य सीएसएस विकल्प
 
-Gatsby supports almost every possible styling option (if there isn't a plugin yet for your favorite CSS option, [please contribute one!](/contributing/how-to-contribute/))
+Gatsby लगभग हर संभव स्टाइलिंग विकल्प का समर्थन करता है (अगर आपके पसंदीदा सीएसएस विकल्प के लिए अभी तक कोई प्लगइन नहीं है, तो [कृपया योगदान दें!](/contributing/how-to-contribute/)
 
 - [Typography.js](/packages/gatsby-plugin-typography/)
 - [Sass](/packages/gatsby-plugin-sass/)
 - [JSS](/packages/gatsby-plugin-jss/)
 - [Stylus](/packages/gatsby-plugin-stylus/)
 - [PostCSS](/packages/gatsby-plugin-postcss/)
 
-and more!
+और अधिक!
```

Comment:
`और अधिक` => `और ऐसे काफी सारे`

I am not very sure about this. What do you think?

Reply:
sounds better

Reply:
how about ```और काफी सारे```

Reply:
> how about `और काफी सारे`

much better. 👍